packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -1561,7 +1561,7 @@ void main() {
     expect(text.style!.fontWeight, FontWeight.w400);
   }, skip: isContextMenuProvidedByPlatform); // [intended] only applies to platforms where we supply the context menu.
 
-  testWidgets('text field toolbar options correctly changes options', (WidgetTester tester) async {
+  testWidgets('text field toolbar options correctly changes options on Apple Platforms', (WidgetTester tester) async {
```

Comment:
FYI I'm deprecating toolbarOptions in my context menu PR.  I worry that these tests might get deleted when toolbarOptions is removed a year from now.  Maybe change the title to indicate that they're testing longpress behavior too?  If you think they should not be deleted.

Reply:
I think we can delete these tests with the deprecation. There are other tests that more implicitly test long press. This test also doesn't test for long press on apple platforms since they do not select a word on long press, only move the cursor.